common-arcana.lic

```diff
@@ -133,6 +142,28 @@ module DRCA
     DRC.retreat(settings.ignored_npcs) unless spell['skip_retreat']
   end
 
+  def prepare_to_cast_runestone(spell, settings)
+    if DRCI.inside?("#{spell['runestone_name']}", settings.runestone_storage)
+      return false if !get_runestone(spell['runestone_name'], settings)
+    else
+      DRC.message("*** Out of #{spell['runestone_name']}! ***")
+      return false
+    end
+    DRC.bput("harness #{settings.runestone_harness}", "You tap into the mana") if settings.runestone_harness
```

Comment:
Is the amount of mana to harness really a global value? I would have expected it to vary based on the spell or skill.

Reply:
Recall, this setting is used for `invoke-runestone` as well. That script did not make use of cambrinth. So, As suggested under usage, do not set this to a value or set to a value you want in both (if using both scripts).

Reply:
`Invoke-runestone` needs harness or other way to put more mana into the spell. Recall, no spell settings are in place if using `invoke-runestone.`

Reply:
I removed from common-arcana since `invoke-runestone` does not use this file.